numpy/core/src/npysort/simd_argsort.dispatch.cpp

```diff
@@ -0,0 +1,44 @@
+/*@targets
```

Comment:
I split this out as Highway's sort doesn't have an implementation of argsort, unless I'm wrong @jan-wassenberg?

Reply:
FYI argsort can be implemented as copying an array to hwy::K32V32 or K64V64 types (putting the key in .key and 0..N-1 into .value) and then copying only the .value back to the array.